### DISCUSSION TITLE
Where is ISO data format setting?

### DISCUSSION THREAD

**Discussion by Drizzt321** [mainsail-crew/mainsail#1813] [Help]
So I'm trying to update the date format, but I don't see an option for ISO date format (YYYY-MM-DD) in the options selection. Seems like an oversight.

**Comment by meteyou** [comment]
you could just change your OS or browser to ISO, then Mainsail would display this as well.

but i think it would be a good idea to add this format as well. Please create a "Feature Request" in the issue section as a reminder for us. This is our ToDo list.

**Reply by Drizzt321** [reply]
How the heck would I change that in the browser? Doesn't seem to be a normal thing...

But good point, didn't see the Feature Request section. 

https://github.com/mainsail-crew/mainsail/issues/1814

**Reply by meteyou** [reply]
ok. i double-checked chrome and firefox. both only load the system/os settings any more.
